spec/mailers/requests/request_mailer_spec.rb

```diff
@@ -1215,7 +1215,7 @@
     # rubocop:disable RSpec/ExampleLength
     it "sends the email and renders the headers and body" do
       mail = Requests::RequestMailer.send("on_shelf_email", submission_for_on_shelf).deliver_now
-      expect(mail.subject).to eq("#{I18n.t('requests.on_shelf.email_subject')} (C) PL2727.S2 C574 1998")
+      expect(mail.subject).to eq("#{I18n.t('requests.on_shelf.email_subject')} (EASTASIAN$CJK) PL2727.S2 C574 1998")
```

Comment:
Are the librarians ok with the pickup locations changing?

Reply:
The codes have literally changed so I'm not sure there is much choice. This is an issue in multiple spots.